web/src/app/views/camp-view/camp-view.component.html

```diff
@@ -17,36 +17,36 @@ <h1>Tábor na ostrově ŠÁN</h1>
         <div class="row text-center">
             <div class="col-12">
                 <h1 class="mb-3">Letní vodácký tábor - ŠÁN 2022</h1>
```

Comment:
Ještě chybí změna roku tady v nadpise, jinak super!

Reply:
A to mi to Máčko ještě kontrolovala jestli je vše ok :-D